integration_tests/tests/test_queries.py

```diff
@@ -26,6 +26,22 @@
                 "subscriber_subset": None,
             },
         ),
+        (
+            "unique_subscribers_count",
+            {
+                "start_date": "2016-01-01",
+                "end_date": "2016-01-02",
+                "aggregation_unit": "admin3",
+            },
+        ),
+        (
+            "unique_subscribers_count",
```

Comment:
Why was this added twice? It seems to be an exact copy of the previous one?

Reply:
DOH! Looking at what I thought was one previous test case, I assumed it was the input arguments and the expected output arguments. But on closer inspection I see it is two previous test cases and only the input arguments are required. I've removed the duplicate entry now and re-committed & pushed to github